packages/core/admin/server/services/api-token.js

```diff
@@ -2,14 +2,18 @@
 
 const crypto = require('crypto');
 
+/**
+ * @typedef {'read-only'|'full-access'} TokenType
+ */
+
 /**
  * @typedef ApiToken
  *
- * @property {number} id
+ * @property {number|string} id
```

Comment:
Why this change?

Reply:
The `id` isn't necessary a number right? IIRC @alexandrebodin pointed that out in my previous PR. When I checked the validation rules for the `id`'s I saw that you have a custom rules that check if it's either a string or a number. So I adapted the jsdoc to reflect that.

Reply:
I just talked with Alex. So, currently in V4 we only have numbers (in V3 we had string to because we had mongoDB, but not anymore). However, we may allow using uuid in the future so it's not a problem to already say it can be a string. It automatically cast anyway if we put `'1'`.
So we can let `number|string` 👍